.github/workflows/cr.yml

```diff
@@ -0,0 +1,36 @@
+name: cr
+on:
+  push:
+    branches: [main]
+    tags: ['!**'] # Avoid publishing on tags
+  pull_request:
+    types: [opened, synchronize, labeled] # Run on PR creation, updates, and when labels are added
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.number }} # Concurrency group for each PR
+  cancel-in-progress: true # Cancel in progress builds for the same PR
+
+jobs:
+  publish:
+    if: github.repository == 'honojs/middleware' && (github.ref == 'refs/heads/main' || contains(github.event.pull_request.labels.*.name, 'cr-tracked'))
```

Comment:
Only run the publish job if the PR has the `cr-tracked` label

Reply:
Ah, using the `cr-tracked` label! I see.